File: tasks/translation/output/Qwen/Qwen2.5-Coder-32B-Instruct-AWQ/codenet/mad/Java/Go/s380356166_comments_removed.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
	"strings"
)

func main() {
	in := bufio.NewReader(os.Stdin)
	out := bufio.NewWriter(os.Stdout)
	defer out.Flush()

	solver := Solver{}
	solver.solve(1, in, out)
}

type Solver struct{}

func (s Solver) solve(testNumber int, in *bufio.Reader, out *bufio.Writer) {
	ans := 0
	line, _ := in.ReadString('\n')
	nums := strings.Fields(line)
	K, _ := strconv.Atoi(nums[0])
	S, _ := strconv.Atoi(nums[1])
	for x := 0; x <= K; x++ {
		for y := 0; y <= K; y++ {
			wk := S - x - y
			if wk >= 0 && wk <= K {
				ans++
			}
		}
	}
	fmt.Fprintln(out, ans)
}

// End of Code

File: tasks/translation/output/Qwen/Qwen2.5-32B-Instruct-AWQ/codenet/mad/Java/Python/s380356166_comments_removed.py
def solve(K, S):
    ans = 0
    for x in range(K + 1):
        for y in range(K + 1):
            wk = S - x - y
            if 0 <= wk <= K:
                ans += 1
    print(ans)

if __name__ == "__main__":
    K = int(input())
    S = int(input())
    solve(K, S)  # End of Code
